apitest/docs/api-beta-test-guide.md

```diff
@@ -0,0 +1,492 @@
+`--tx-fee-rate=<sats/byte>` options override both the default network fee rate, and your custom transaction fee
+setting for the execution of those commands.
+
+#### Using Default Network Transaction Fee
+
+If you have not set your custom withdrawal transaction fee setting, the default network transaction fee will be used
+when withdrawing funds.  In either case, you can check the current (default or custom) withdrawal transaction fee rate:
+```
+$ ./bisq-cli --password=xyz gettxfeerate
+```
+#### Setting Custom Transaction Fee Preference
+To set a custom withdrawal transaction fee rate preference of 50 sats/byte:
+```
+$ ./bisq-cli --password=xyz settxfeerate --tx-fee-rate=50
+```
+
+#### Removing User’s Custom Transaction Fee Preference
+To remove a custom withdrawal transaction fee rate preference, and revert to the network fee rate:
+```
+$ ./bisq-cli --password=xyz unsettxfeerate
+```
+
+### Creating Test Payment Accounts
+
+Creating a payment account using the Api involves three steps:
+
+1.  Find the payment-method-id  for the payment account type you wish to create.  For example, if you want to
+    create a face-to-face type payment account, find the face-to-face  payment-method-id (`F2F`):
+    ```
+    $ ./bisq-cli --password=xyz --port=9998 getpaymentmethods
+    ```
+
+2.  Use the payment-method-id `F2F` found in the `getpaymentmethods` command output to create a blank payment account
+    form:
+    ```
+    $ ./bisq-cli --password=xyz --port=9998 getpaymentacctform --payment-method-id=F2F
+    ```
+    This `getpaymentacctform` command generates a json file (form) for creating an `F2F` payment account,
+    prints the file’s contents, and tells you where it is.  In this example, the sever created an `F2F` account
+    form named `f2f_1612381824625.json`.
+
+
+3.  Manually edit the json file, and use its path in the `createpaymentacct` command.
+    ```
+    $ ./bisq-cli --password=xyz --port=9998 createpaymentacct
+        --payment-account-form=f2f_1612381824625.json
+    ```
+    The server will create and save the new payment account from details defined in the json file.
+
+    _Note:  You can rename the file before passing it to the  `createpaymentacct` command._
+
+### Creating Offers
+
+The createoffer command is the Api's most complex command (so far), but CLI posix-style options are self-explanatory,
+and CLI `createoffer` command help gives you specific information about each option.
+```
+$ ./bisq-cli --password=xyz --port=9998 createoffer --help
+```
+
+#### Examples
+
+The `trade-simulation.sh` script described above is an easy way to figure out how to use this command.
+In one of the examples, Alice created a BUY/ EUR offer to buy 0.125 BTC at a fixed price of  30,800 EUR,
+and pay the Bisq maker fee in BSQ.  Alice had already created an EUR face-to-face payment account with id
+`f3c1ec8b-9761-458d-b13d-9039c6892413`, and used this `createoffer` command:
```

Comment:
How does one know this payment account ID? Does `createpaymentacct` return it upon successful account creation?

Reply:
Yes, `createpaymentacct` returns the account ID, so does `getpaymentaccts`.

Reply:
I added an explanation in this section (commit cdf15676dbbcffe7aa383e9c9b7d872f9cde3eb0).